cmd/traffic_manager/traffic_manager.cc

```diff
@@ -740,16 +742,17 @@ main(int argc, const char **argv)
     }
 
     if (lmgmt->run_proxy && !lmgmt->processRunning() && lmgmt->proxy_recoverable) { /* Make sure we still have a proxy up */
-      if (sleep_time) {
+      const uint64_t now = static_cast<uint64_t>(time(nullptr));
+      if (sleep_time && now - last_start_epoc_s < MAX_SLEEP_S) {
```

Comment:
I know amc disagrees, but I think that this would be much more readable like this:

```
if (sleep_time && ((now - last_start_epoc_s) < MAX_SLEEP_S))
```

But, I'm not going to bikeshed / -1 because of this. It's just a coding preference of mine. The point is that when someone else modifies / or looks at this code, there's more risk that they mess it up.

Reply:
Addressed, and for what it's worth, I agree.